bazel/repositories.bzl

```diff
@@ -813,7 +813,6 @@ def _com_github_google_quiche():
         name = "com_github_google_quiche",
         genrule_cmd_file = "@envoy//bazel/external:quiche.genrule_cmd",
         build_file = "@envoy//bazel/external:quiche.BUILD",
-        patches = ["@envoy//bazel/external:quiche.patch"],
```

Comment:
Oh, NICE!!

Reply:
I don't think this is going to work. Are there some patch hunks that are failing to apply?

Reply:
Unfortunately, I have to bring it back.